aiven-go/main.go

```diff
@@ -8,12 +8,12 @@ import (
 func main() {
 	pulumi.Run(func(ctx *pulumi.Context) error {
 		kafka, err := aiven.NewKafka(ctx, "aiven-kafka", &aiven.KafkaArgs{
```

Comment:
Similar comments here as those provided on `aiven-python` and `aiven-typescript`.